src/zenml/models/v2/misc/server_models.py

```diff
@@ -73,6 +73,14 @@ class ServerModel(BaseModel):
     auth_scheme: AuthScheme = Field(
         title="The authentication scheme that the server is using.",
     )
+    base_url: str = Field(
+        "",
+        title="The Base URL of the server.",
+    )
+    metadata: dict = Field(
+        {},
+        title="The metadata associated with the server.",
+    )
```

Comment:
```suggestion
    metadata: Dict[str, str] = Field(
        default_factory=dict,
        title="The metadata associated with the server.",
    )
```

and you probably need to import Dict from typing too